app/src/lib/stores/app-store.ts

```diff
@@ -514,6 +516,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
   private useWindowsOpenSSH: boolean = false
 
+  private showCommitLengthWarning: boolean = false
```

Comment:
```suggestion
  private showCommitLengthWarning: boolean = showCommitLengthWarningDefault
```

Go ahead and use the default.